VologramsToolkit/Scripts/Player/VolPlayer.cs

```diff
@@ -614,33 +609,37 @@ private void ReadNextGeom(int frame = -1)
             _meshFilter.mesh.SetNormals(normalsSlice.ToArray());
 #endif
         }
-
-        if (_geometryData.indicesSize > 0)
+        // TODO(Anton) - Patrick please check this, I was hoping to skip some updates here
+        isKeyframe = true; // HACK(Anton) ... but it didn't work.
+ 

```

Comment:
i need a hand here - this is the bit i referred to in the PR description. i assumed i could just not upload indices/uvs if it wasnt a keyframe but it seems to need that. perhaps there's a clear() somewhere that is resetting things.
i wanted to reduce mesh upload bandwidth - not a big deal but should be possible to get a tiny performance improvement.

Reply:
On line 588/590 there is a mesh clear. There was an issue about updating mesh components without clearing in runtime, but I can't remember what it was

Reply:
okay. we can put that in a ticket for a potential follow-up